connector-j-5/src/main/java/com/google/cloud/sql/mysql/SocketFactory.java

```diff
@@ -84,14 +85,15 @@ public Socket afterHandshake() {
 
   // TODO(kurtisvg) move this check into a shared class
   // Returns @{code true} if running in a Google App Engine Standard runtime.
-  private boolean runningOnGaeStandard(){
+  private boolean runningOnGaeStandard() {
     // gaeEnv="standard" indicates standard instances
     String gaeEnv = System.getenv("GAE_ENV");
     // runEnv="Production" requires to rule out Java 8 emulated environments
     String runEnv = System.getProperty("com.google.appengine.runtime.environment");
     // gaeRuntime="java11" in Java 11 environments (no emulated environments)
     String gaeRuntime = System.getenv("GAE_RUNTIME");
 
-    return "standard".equals(gaeEnv) && ("Production".equals(runEnv) || "java11".equals(gaeRuntime));
+    return "standard".equals(gaeEnv) && ("Production".equals(runEnv) || "java11"
```

Comment:
This doesn't look like it was formatted by [google-java-format](https://github.com/google/google-java-format). That's available as an IntelliJ plugin as well as a command-line tool.

Generally we apply it in the mode where it only affects lines that were already touched in a PR, in order to avoid extraneous changes, unless there's a reformat-only PR. But it's up to you.

Reply:
Ok, I believe I've run the correct one now. (I apparently only had it enabled for g3 files)